src/library/playlisttablemodel.cpp

```diff
@@ -255,6 +255,10 @@ TrackModel::CapabilitiesFlags PlaylistTableModel::getCapabilities() const {
     if (m_iPlaylistId != m_pTrackCollection->getPlaylistDAO().getPlaylistIdFromName(AUTODJ_TABLE)) {
         caps |= TRACKMODELCAPS_ADDTOAUTODJ;
     }
+	// Disable reording tracks for history playlists
+	if (m_pTrackCollection->getPlaylistDAO().getHiddenType(m_iPlaylistId)== PlaylistDAO::PLHT_SET_LOG) {
+         caps &= ~TRACKMODELCAPS_REORDER;
```

Comment:
Nit: Please replace tabs by four spaces.

Reply:
Hm that is odd, my editor should be taking care of this. I will have to check this out. Thanks for the notice.